content/docs/buildpacks/language-family-buildpacks/java.md

```diff
@@ -164,39 +176,79 @@ pack inspect-image samples/app --bom | jq '.local[] | select(.name=="jre") | .me
 {{< /code/copyable >}}
 
 #### Configuring the JVM Version
+
 The following environment variable configures the JVM version at build-time.
+
 * `BP_JVM_VERSION`
-    * Defaults to the latest LTS version at the time of release.
-    * Configures a specific JDK or JRE version.
-    * *Example*: Given `BP_JVM_VERSION=8` or `BP_JVM_VERSION=8.*` the buildpack will install the latest patch releases of the Java 8 JDK and JRE.
+  * Defaults to the latest LTS version at the time of release.
+  * Configures a specific JDK or JRE version.
+  * *Example*: Given `BP_JVM_VERSION=8` or `BP_JVM_VERSION=8.*` the buildpack will install the latest patch releases of the Java 8 JDK and JRE.
 
 ### Runtime JVM Configuration
+
 The Java Buildpack configures the JVM by setting `JAVA_TOOL_OPTIONS` in the JVM environment.
- 
+
 #### Memory Calculator
+
 The Java Buildpack installs a component called the Memory Calculator which will configure JVM memory based on the resources available to the container at runtime. The calculated flags will be appended to `JAVA_TOOL_OPTIONS`.
 
 #### Configuring JVM at Runtime
+
 The runtime JVM can be configured in two ways:
-1. Buildpack-provided runtime components including the Memory Calculator accept semantically named environment variables which are then used to derive `JAVA_TOOL_OPTIONS` flags. Examples include: 
+
+1. Buildpack-provided runtime components including the Memory Calculator accept semantically named environment variables which are then used to derive `JAVA_TOOL_OPTIONS` flags. Examples include:
     * `BPL_JVM_HEAD_ROOM`
     * `BPL_JVM_LOADED_CLASS_COUNT`
     * `BPL_JVM_THREAD_COUNT`
 2. Flags can be set directly at runtime with the `JAVA_TOOL_OPTIONS` environment variable. User-provided flags will be appended to buildpack-provided flags. If the user and a buildpack set the same flag, user-provided flags take precedence.
 
 See the [homepage][bp/bellsoft-liberica] for the Bellsoft Liberica Buildpack for a full set of configuration options.
 
+### Using Alternative JVM Vendors
+
+By default, the [Paketo Java buildpack][bp/java] will use the Liberica JVM. The following alternative JVM vendors may be substituted in place of Liberica's JVM.
+
+* [Paketo Amazon Corretto Buildpack][bp/amazon-corretto]
+* [Paketo Azul Zulu Buildpack][bp/azul-zulu]
+* [Paketo BellSoft Liberica Buildpack - Default][bp/bellsoft-liberica]
+* [Paketo Eclipse OpenJ9 Buildpack][bp/eclipse-openj9]
+* [Paketo GraalVM Buildpack][bp/graalvm]
+* [Paketo SapMachine Buildpack][bp/sap-machine]
+
+To use an alternative JVM vendor, you will need to set two `--buildpack` arguments to `pack build`, one for the alternative JVM vendor you'd like to use and one for the Paketo Java buildpack (in that order). This works because while you end up with two JVM vendor buildpacks, the first one, the one you're specifying will claim the buildplan entries so the second one will end up being a noop and doing nothing.
```

Comment:
```suggestion
To use an alternative JVM, you will need to set two `--buildpack` arguments to `pack build`, one for the alternative JVM buildpack you'd like to use and one for the Paketo Java buildpack (in that order). This works because while you end up with two JVM buildpacks, the first one, the one you're specifying will claim the build plan entries so the second one will end up being a noop and doing nothing.
```